src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -919,15 +926,30 @@ const Authenticate = (() => {
     };
 
     const initAuthentication = async () => {
+        let has_personal_details_error = false;
         const authentication_status = await getAuthenticationStatus();
 
         if (!authentication_status || authentication_status.error) {
             $('#authentication_tab').setVisibility(0);
             $('#error_occured').setVisibility(1);
             return;
         }
+
+        const service_token_response = await getOnfidoServiceToken();
+        
+        if (
+            service_token_response.error &&
+            service_token_response.error.code === 'MissingPersonalDetails'
+        ) {
+            has_personal_details_error = true;
+            let missing_personal_fields = '';
+            Object.keys(service_token_response.required_fields).forEach(field => {
+                missing_personal_fields += !missing_personal_fields ? personal_details_map[field] : `, ${personal_details_map[field]}`;
+            });
```

Comment:
@mustofa-binary please have a look at how we do it in the existing files https://github.com/binary-com/deriv-app/blob/dev/packages/core/src/Modules/Cashier/Components/Error/error.jsx#L121

better to stick to one way of doing it